change/@itwin-eslint-plugin-34f477fe-e0c8-4c40-82ae-72b50e0cf166.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
+  "comment": "Updated vulnerable dependencies",
```

Comment:
I would prefer to remove this change file and just deploy the updated version ranges with the next release, whenever that happens for other reasons. As I said, this has no effect on users so there's no reason to bump the version just for this.